examples/snippets/simple_app.py

```diff
@@ -7,10 +7,10 @@
 import asfquart.session
 
 
-def my_app():
+def my_app() -> asfquart.base.QuartApp:
     # Construct the base app. The /oauth gateway is enabled by default.
     # To disable it, use construct("my_app", oauth=False)
-    app = asfquart.construct("my_simple_app")
+    app = asfquart.construct("my_simple_app", cfg_file="config.yaml")
```

Comment:
No need to specify cfg_file, as that's already the default.

Reply:
that is something that I forget to remove while testing the settings.